ratpack-rx/src/test/groovy/ratpack/rx/RxErrorHandlingSpec.groovy

```diff
@@ -200,7 +200,7 @@ class RxErrorHandlingSpec extends RatpackGroovyDslSpec {
     thrownException == e
   }
 
-  def "on complete can throw"() {
+  def "on complete cannot throw"() {
```

Comment:
I think this is required due to a change in behaviour in RxJava (https://github.com/ReactiveX/RxJava/pull/3155)